client/src/components/NavBar.jsx

```diff
@@ -1,20 +1,40 @@
 //Import necessary functionalities
-import React, { useContext } from 'react';
-import ComponentContext from '../ComponentContext.js';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { UserStatus } from '../context/UserContext.jsx';
 
-const NavBar = ({ }) => {
+//Import log in component to hang out in the upper right hand corner of the page
+import LogIn from './LogIn.jsx';
 
-    const { setComponent } = useContext(ComponentContext);
+const NavBar = () => {
+
+    //Import user information to toggle log in/log out buttons
+    const { loggedUser, setLoggedUser } = UserStatus();
+
+    //Logout function to remove logged user
+    const logout = () => {
+        setLoggedUser(null);
+    }
 
     return (
         <nav>
             {/* Replace header with logo in the future */}
-            <h1 onClick={() => setComponent('homepage')}>Click and Treat</h1>
+            {/* <h1 onClick={() => loggedUser ? setComponent('dashboard') : setComponent('homepage')}>Click and Treat</h1> */}
+            {/* Need to create function to link to homepage or dashboard based on loggedUser */}
+            
+            {loggedUser
+            ? <Link to="/dashboard">Click and Treat</Link>
+            : <Link to="/homepage">Click and Treat</Link>}
```

Comment:
This made me actually laugh out loud because these comments should've been removed ages ago! So sorry for the confusion but I totally see how wacky that sounded with the functionality already implemented. Those are left over from when I was using useContext. Thank you for catching that!!! I have removed them in commit [3ec6c70](https://github.com/tati-guzman/click-and-treat/pull/3/commits/3ec6c70375055dfff349fd24d736ff4a2b260625).